streaming/java/streaming-api/src/main/java/org/ray/streaming/api/function/impl/FlatMapFunction.java

```diff
@@ -4,7 +4,7 @@
 import org.ray.streaming.api.function.Function;
 
 /**
- * Interface of flat-map functions.
+ * Interface of flat-MAP functions.
```

Comment:
ditto

Reply:
Thanks for replying. I'll check the codes.